src/iop/temperature.c

```diff
@@ -491,6 +532,12 @@ int process_cl(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, cl_m
   cl_int err = -999;
   int kernel = -1;
 
+  if(img->flags & DT_IMAGE_4BAYER)
+  {
+    dt_print(DT_DEBUG_OPENCL, "[opencl_temperature] temperature for CYGM not yet supported by opencl code\n");
+    return FALSE;
```

Comment:
Now this is completely wrong.
Look what is being done for X-Trans in commit_params() and do the same.